Registry/RegistryParse.py

```diff
@@ -1047,10 +1047,10 @@ def data(self):
         """
         data_type = self.data_type()
         data_length = self.raw_data_length()
-        d = self.raw_data()
+        d = self.raw_data(overrun)
```

Comment:
please use `overrun=overrun` here just to make it explicit what argument this is